doc/api/esm.md

```diff
@@ -744,56 +788,58 @@ Node.js module specifier resolution behavior_ when calling `defaultResolve`, the
 `context.conditions` array originally passed into the `resolve` hook.
 
 ```js
-/**
- * @param {string} specifier
- * @param {{
- *   conditions: string[],
- *   parentURL: string | undefined,
- * }} context
- * @param {Function} defaultResolve
- * @returns {Promise<{ url: string }>}
- */
-export async function resolve(specifier, context, defaultResolve) {
+export async function resolve(specifier, context, nextResolve) {
   const { parentURL = null } = context;
+
   if (Math.random() > 0.5) { // Some condition.
     // For some or all specifiers, do some custom logic for resolving.
     // Always return an object of the form {url: <string>}.
     return {
+      shortCircuit: true,
       url: parentURL ?
         new URL(specifier, parentURL).href :
         new URL(specifier).href,
     };
   }
+
   if (Math.random() < 0.5) { // Another condition.
     // When calling `defaultResolve`, the arguments can be modified. In this
     // case it's adding another value for matching conditional exports.
-    return defaultResolve(specifier, {
+    return nextResolve(specifier, {
       ...context,
       conditions: [...context.conditions, 'another-condition'],
     });
   }
-  // Defer to Node.js for all other specifiers.
-  return defaultResolve(specifier, context, defaultResolve);
+
+  // Defer to the next hook in the chain, which would be the
+  // Node.js default resolve if this is the last user-specified loader.
+  return nextResolve(specifier, context);
 }
 ```
 
-#### `load(url, context, defaultLoad)`
+#### `load(url, context, nextLoad)`
 
```

Comment:
There are a few places that mention this, and they all use `shortCircuit: true`. Mind if we use that here too? (I think it's more straightforward, and concise—like "copy+paste this, and you're good")

Reply:
Sure, my point was we need a history section that list the changes, no strong opinion on how to word it best, I'm happy to leave that out to you :)

Reply:
Actually, there is already an entry for this under the `## Loaders` heading. Since it applies to both and was introduced at the same time, would it not be appropriate to consolidate it there?